.buildkite/pipeline-raw.yml

```diff
@@ -2,31 +2,32 @@
 # SPDX-License-Identifier: LicenseRef-MIT-OA
 
 env:
-  SET_VERSION: "export TEZOS_VERSION=\"$(cat nix/nix/sources.json | jq -r '.tezos.ref' | cut -d'/' -f3)\""
+  SET_VERSION: "export TEZOS_VERSION=\"$(cat meta.json | jq -r '.tezos_ref' | cut -d'/' -f3)\""
+  USE_NEWER_NIX: 1
 
 steps:
  # We need to sign commits that update brew formulae separately
  - label: Sign formulae update commits
    if: build.branch =~ /^auto\/update-brew-formulae-.*/
    commands:
-   - nix-shell ./scripts/shell.nix
-       --run './scripts/sign-commits.sh'
+   - nix develop .#buildkite
+       --command './scripts/sign-commits.sh'
  # To avoid race-conditions for the gpg key between jobs which sometimes leads to weird errors
  - wait
 
  - label: reuse lint
-   command: nix run -f . pkgs.reuse -c reuse lint
+   command: nix shell .#reuse -c reuse lint
  - label: check trailing whitespace
    command: .buildkite/check-trailing-whitespace.sh
  - label: crossref-verify
-   command: "nix run -f https://github.com/serokell/crossref-verifier/archive/68a1f9d25b6e7835fea8299b18a3e6c61dbb2a5c.tar.gz -c crossref-verify"
+   command: "USE_NEWER_NIX=0; nix run -f https://github.com/serokell/crossref-verifier/archive/68a1f9d25b6e7835fea8299b18a3e6c61dbb2a5c.tar.gz -c crossref-verify"
```

Comment:
While we're at it, could you please update this outdated piece of CI pipeline as well? Apparently, [xrefcheck](https://github.com/serokell/xrefcheck) should be added as another flake input and used here instead of `nix run`ing specific revision of old repo